<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>590</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>590</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>820</width>
    <height>590</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Калькулятор</string>
       </attribute>
       <widget class="QLineEdit" name="input">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>551</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 25pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="output">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>100</y>
          <width>231</width>
          <height>41</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="equalsButton">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>20</y>
          <width>101</width>
          <height>61</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"> font: 30pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>=</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton6">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>300</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>6</string>
        </property>
       </widget>
       <widget class="QPushButton" name="atanButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>400</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>atan</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton9">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>380</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>9</string>
        </property>
       </widget>
       <widget class="QPushButton" name="asinButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>400</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>asin</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton7">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>380</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>7</string>
        </property>
       </widget>
       <widget class="QPushButton" name="tanButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>340</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>tan</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>220</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>2</string>
        </property>
       </widget>
       <widget class="QPushButton" name="logButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>460</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>log</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bracketRightButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>280</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>)</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>380</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>8</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton0">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>460</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>220</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sinButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>340</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>sin</string>
        </property>
       </widget>
       <widget class="QPushButton" name="powButton">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>280</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>^</string>
        </property>
       </widget>
       <widget class="QPushButton" name="sqrtButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>400</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>sqrt</string>
        </property>
       </widget>
       <widget class="QPushButton" name="bracetLeftButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>280</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>(</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>460</y>
          <width>41</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>X = </string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>300</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>5</string>
        </property>
       </widget>
       <widget class="QPushButton" name="modButton">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>280</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>mod</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="inputX">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>460</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="mulButton">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>220</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>*</string>
        </property>
       </widget>
       <widget class="QPushButton" name="acosButton">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>340</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>acos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cosButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>340</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>cos</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>4</string>
        </property>
       </widget>
       <widget class="QPushButton" name="numButton3">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>220</y>
          <width>61</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 24pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>3</string>
        </property>
       </widget>
       <widget class="QPushButton" name="lnButton">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>400</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>ln</string>
        </property>
       </widget>
       <widget class="QPushButton" name="divButton">
        <property name="geometry">
         <rect>
          <x>700</x>
          <y>220</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
       <widget class="QPushButton" name="subButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>220</y>
          <width>80</width>
          <height>40</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 18pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>График</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>181</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Xmin </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Xmax</string>
           </property>
           <property name="textFormat">
            <enum>Qt::MarkdownText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="XminLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="XmaxLineEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="inputFunc">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>481</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 30pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>280</y>
          <width>181</width>
          <height>151</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Ymin </string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Ymax</string>
           </property>
           <property name="textFormat">
            <enum>Qt::MarkdownText</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="YminLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="YmaxLineEdit"/>
         </item>
        </layout>
       </widget>
       <widget class="QCustomPlot" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>289</x>
          <y>99</y>
          <width>471</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="plotPutton">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>440</y>
          <width>181</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Кредитный</string>
       </attribute>
       <widget class="QLineEdit" name="amountInput">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Сумма кредита</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="termInput">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Количество месяцев</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>170</y>
          <width>58</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Процент</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="rateInput">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>113</width>
          <height>31</height>
         </rect>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
       <widget class="QGroupBox" name="typeCreditGB">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>230</y>
          <width>171</width>
          <height>80</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 13pt &quot;.AppleSystemUIFont&quot;;</string>
        </property>
        <property name="title">
         <string>Тип кредита</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>171</width>
           <height>61</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="annuity">
            <property name="text">
             <string>аутифицированный</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <property name="autoRepeat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="diff">
            <property name="text">
             <string>Дифференцированный</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QPushButton" name="CreditCalcButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>321</y>
          <width>121</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Расчитать</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="totalOut">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>40</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="overpaymentOut">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>90</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="paymentOut">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>140</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>50</y>
          <width>121</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Итоговая выплата</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>100</y>
          <width>111</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Переплата</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>150</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ежемесячный платёж</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="diffPayWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>140</y>
          <width>241</width>
          <height>241</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Месяц</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Платёж</string>
         </property>
        </column>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Депозитный</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
